*** tests/A-004-004-M-ok.out	Fri Oct  6 12:23:10 2023
--- tests/A-004-004-M-ok.outhyp	Wed Oct 11 13:42:04 2023
***************
*** 24,31 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;3|7
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 24,30 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
